### DISCUSSION TITLE
Afra Discussion

### DISCUSSION THREAD

**Discussion by itafra-net** [IT-Afra/bakery-shop#61] [Announcements]
## 👋 Welcome To Afra Discussion !

Our goal in this discussion is to make progress on the project, enhance its performance, and also fix the problems in the project.

Therefore, we appreciate and thank those who can accompany us in this matter.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mohsenarg** [comment]
Thank you very much, Afra.

I hope that I can help you in the progress of this project

**Comment by mohammadpirzadeh** [comment]
Hello Afra
Thanks for this opportunity
I hope we can achieve the best together
